bin/sdk-cli.js

```diff
@@ -125,7 +128,7 @@ yargs
 	.command( {
 		command: 'generic <entry-point> <output-name>',
 		desc: 'Build generic JavaScript code',
-		builder: yargs => yargs
+		builder: () => yargs
```

Comment:
I assume this was prompted by eslint, but it's not synonymous.

It's probably simplest to leave these `yargs => yargs…` alone an rename the module import `const yargsModule = require( 'yargs' )` and rename at the top level.

Reply:
Valid point; it _seems_ to work but fair enough to play safe here.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/29319/commits/914e5c1557baec35b680578fffc150516500b193